### DISCUSSION TITLE
Support reviewing commit messages

### DISCUSSION THREAD

**Discussion by xmo-odoo** [community/community#31741] [Pull Requests] [answered]
edit: dupe of #4983

Many projects consider good commit messages to be critical to their long-term health: as they are embedded in the repository itself, they are a permanent record of intent and decisions, immune to vagaries like change of hosting platform, maintainer, forking (unless the fork is so aggressive / contentious it decides to reset history but nothing can survive that), etc...

However Github provides basically no tools to review commit messages of PRs, all a maintainer can do (aside from editing the commits themselves, if the contributor allows it) is provide vague instructions PR comments, with just as vague references to the commit whose message is being mentioned.

This is a far cry from the pretty extensive commenting abilities on the PR's changes.

**Comment by bananartist** [comment, marked as answer]
Every day I which this feature existed. This discussion is a duplicate of https://github.com/orgs/community/discussions/4983

**Reply by xmo-odoo** [reply]
Oh hey one more issue I completely forgot I'd already found and upvoted, and reopened a thread for, thanks for the link.

I can't close this, but if someone allowed to comes along, this can be closed as a dupe.
